.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```

Comment:
Please don't include files in the project which are not related to the project.

Reply:
it should be deleted before